app/retail/templates/emails/grants/transaction_summary.html

```diff
@@ -0,0 +1,234 @@
+{% extends '../emails/template.html' %}
+{% comment %}
+  Copyright (C) 2018 Gitcoin Core
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published
+  by the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n static %}
+
+{% block content %}
+
+<style>
+  hr {
+    width: 80%;
```

Comment:
should some of these be in a shared template?  they look repeated

Reply:
You mean move the css into standalone template file ? 🤔 
as in it does seem a lil weird but we could do that

Reply:
Yes, could be in a shared template. Will put refactor onto list.